core/data_migration_scripts/4_1_0_stages/EE_DMS_4_1_0_event_venue.dmsstage.php

```diff
@@ -53,7 +62,11 @@ public function _migration_step($num_items = 50)
     public function _count_records_to_migrate()
     {
         global $wpdb;
-        $count = $wpdb->get_var("SELECT COUNT(id) FROM ".$this->_old_table);
+        $count = $wpdb->get_var(
+            "SELECT COUNT(ev.id) FROM ".$this->_old_table . " AS ev 
+            INNER JOIN " . $wpdb->prefix . 'events_detail AS e ON ev.event_id=e.id
```

Comment:
here you have `e.id` and `ev.event_id` yet in the select part of the query you are using `COUNT(ev.id)`.  Which should it be `COUNT(e.id)` or `COUNT(ev.event_id)`?

Reply:
Hmmm, thanks for following the code along this well.
`$this->_old_table` was the EE3 `events_venue_rel` table, so previously we were counting `events_venue_rel.id`.
With this change, we're also joininig to the events table so we can skip event-venue-relation rows for deleted events, but I think we should continue counting the same column, `events_venue_rel.id`, aliased as `ev.id`.
Does that make sense?

Reply:
I think I understand.  Essentially `ev.id` then is the event_venue relation id not the event_id.  If that's a correct understanding then ya this is fine.